pkgs/epnix/tools/phoebus/olog/default.nix

```diff
@@ -8,16 +8,16 @@
 }:
 maven.buildMavenPackage rec {
   pname = "phoebus-olog";
-  version = "4.7.3";
+  version = "4.7.6";
```

Comment:
It seems phoebus-olog released [v4.7.7](https://github.com/Olog/phoebus-olog/tags). Could you update your PR?